goverance.md

```diff
@@ -0,0 +1,67 @@
+
+Technical Leads will lead larger streams of work that require sustained technical leadership. Any new proposal should have a TL working as an active sponsor of the proposal, guiding the proposal with the engagement of community. The general list of activities for TL are
+  - Establish new subprojects
+  - Decommission existing subprojects
+  - Resolve cross-subproject technical issues and decisions
+
+There will be two to three TLs and their membership will be tracked in [`wg.yaml`](). TLs are elected by one of the chairs nominating a candidate and creating a PR that adds the TL to [`wg.yaml`](). Members are invited to comment and vote according to the regular proposal process described below.
+
+## Creating, discussing and accepting proposals
+
+Each proposal is unique and might deviate slightly from the process below. For example, a smaller project may each step. For example, a small addition may not require establishing completion criteria. In general, we encourage this process below to be followed to ensure that contributions are in line with the goals of the Working Group.
+### Proposal process
+
+1. **Raise an Issue.** [Create an issue](https://github.com/cn-security/safe/issues/new) in the SAFE repo that outlines the problem to be solved. If possible, include:
+  * The customer impact of the problem
+  * The scope of the work required
+
+1. **Ask the group for collaboration.** Rather than immediately beginning work on a solution, at the next meeting, bring the issue up for discussion, explain it, and ask if others are interested in collaborating. This ensures that solutions are created with multiple perspectives. The outcome of this conversation will be:
+  * The group agrees that this is a problem to solve
+  * There are criteria for completion or failure, and those will be added to the issue.
+  * At least one person is tasked with creating a solution
+  * Those interested in working on the solution either begin work or set up time or expectations with others to begin work.
+  * A TL is agrees to actively sponsor the work.
+
+1. **Track progress.** As long as work is ongoing, progress should be tracked in the issue.   
+  * Collaborators should update the body of the Issue or PR to include a checklist of work that shows what has been done and what work remains.
+  * Additionally, when possible, someone working on the project should attend the weekly meetings and give an update.
+
+1. **Discuss the work at a meeting.** If an objection to a PR is made either in a comment of the PR or in call discussing the meeting, both the person making the objection and the person making the person making the proposal will have time to present their case at the next meeting. If there are not unaddressed concerns, and the Pull Request has been stable for 48 hours, a Chair will add it to the agenda for an upcoming meeting. Ideally, members who contributed to the project will attend that meeting to present their work or answer questions.
```

Comment:
"the person making the person making the" => "the person making the"

Reply:
Made this change